src/libraries/System.Net.HttpListener/src/System/Net/HttpListenerResponse.cs

```diff
@@ -79,20 +79,11 @@ public bool SendChunked
             set => EntitySendFormat = value ? EntitySendFormat.Chunked : EntitySendFormat.ContentLength;
         }
 
-        // We MUST NOT send message-body when we send responses with these Status codes
-        private static readonly int[] s_noResponseBody = { 100, 101, 204, 205, 304 };
 
-        private static bool CanSendResponseBody(int responseCode)
-        {
-            for (int i = 0; i < s_noResponseBody.Length; i++)
-            {
-                if (responseCode == s_noResponseBody[i])
-                {
-                    return false;
-                }
-            }
-            return true;
-        }
+
+        private static bool CanSendResponseBody(int responseCode) =>
```

Comment:
It might be worth extracting changes likes this one from this PR

Reply:
Good suggestion.  Done in https://github.com/dotnet/runtime/pull/72727.